### PR TITLE
temp upgrade scripts

### PR DESCRIPTION
**Pull request by Defcon023**
temporary scripts

### CODE REVIEW

**Review by serejke — APPROVED** [review]

File: src/InterchainMessaging/gasPayment/InterchainGasPaymaster.sol
[the remark this answers is not in the document]
Comment (Defcon023) [reply src/InterchainMessaging/gasPayment/InterchainGasPaymaster.sol:1]: Added a todo in the contract

File: script/DeployKhalani.s.sol [hunk 1]
```diff
@@ -50,8 +50,9 @@ contract DeployKhalani is Script {
         //make facet cuts - addKhalaniFacets() -> BridgeFacet, RemoteRequestProcessor, KhalaniSetter
         (khalaniContracts.bridgeFacet, khalaniContracts.requestProcessorFacet, khalaniContracts.setterFacet) = LibDiamondDeployer.addKhalaniFacets(diamondProxy);
 
+        address hyperlaneIGP; //TODO, adjust after finalizing M2 Deployment
```
[the remark this answers is not in the document]
Comment (Defcon023) [reply script/DeployKhalani.s.sol:53]: using IGP now

File: lib/hyperlane-monorepo/solidity/contracts/igps/InterchainGasPaymaster.sol [hunk 2]
```diff
@@ -17,7 +17,7 @@ contract InterchainGasPaymaster is
     IGasOracle,
     OwnableUpgradeable
 {
-    // ============ Constants ============
+    // ==========a== Constants ============
```
[the remark this answers is not in the document]
Comment (samtvlabs) [reply lib/hyperlane-monorepo/solidity/contracts/igps/InterchainGasPaymaster.sol:20]: @Defcon023 can you please fix this

**Review by samtvlabs — CHANGES_REQUESTED** [review]

File: lib/hyperlane-monorepo/solidity/contracts/igps/InterchainGasPaymaster.sol (hunk 2 above)
[the remark this answers is not in the document]
Comment (Defcon023) [reply lib/hyperlane-monorepo/solidity/contracts/igps/InterchainGasPaymaster.sol:20]: https://github.com/tvl-labs/khalani-core/pull/212/commits/6eb5131552c51c6348543a574fa6cea259fb70f8#diff-2a44663ba321cb67cdfa8696ec59cf75b18a95725d911049b6713a76491580e6R21

fixed it here, don't know why its showing old version

[the remark this answers is not in the document]
Comment (samtvlabs) [reply lib/hyperlane-monorepo/solidity/contracts/igps/InterchainGasPaymaster.sol:20]: strange. Thanks!

**Review by samtvlabs — APPROVED** [review]

**Pull request merged**